Mathlib/FieldTheory/PurelyInseparable.lean

```diff
@@ -878,6 +886,7 @@ theorem finSepDegree_eq [Algebra.IsAlgebraic F E] :
 
 /-- The finite separable degree multiply by the finite inseparable degree is equal
 to the (finite) field extension degree. -/
+@[stacks 030L "Part 3, `finDegree` variant"]
```

Comment:
```suggestion
@[stacks 030L "Part 3, `ℕ`-valued variant"]
```

I don't thinks this tag is right.

Reply:
Removed. It is the paragraph following 030L.